README.md

```diff
@@ -8,13 +8,15 @@ This guide will walk you through the steps of creating a **virtual meeting room
 
 To use the Kaltura API, you'll need a Kaltura account and credentials. The important credentials can be found in the [Integration Settings]( https://kmc.kaltura.com/index.php/kmcng/settings/integrationSettings) in your KMC. 
 
-You might want to download a [Kaltura Client Library](https://developer.kaltura.com/api-docs/Client_Libraries/) of your choice, although you can also use the [developer console](https://developer.kaltura.com/console) for all the operations mentioned below. 
+You might want to download a [Kaltura Client Library](https://developer.kaltura.com/api-docs/Client_Libraries/) of your choice, although you can also use the [developer console](https://developer.kaltura.com/console) for all the operations  below. 
 
 This integration will be using your KAF endpoint, which should contain your Partner ID. If you're not sure whether you have a KAF endpoint or whether "Newrow" has been enabled on your account, speak to your account rep or email us at vpaas@kaltura.com.
 
 ### Overview
 
-A virtual meeting room is represented as a `KalturaLocationScheduleResource`. A scheduled  session is represented by a `KalturaRecordScheduleEvent`. One resource can be used for many different events, but an event will only be associated with one resource. The room will be launched using an **embed link that is made up of a Kaltura Session with specific configurations**, all of which will be discussed below. 
+A virtual meeting room is represented as a `KalturaLocationScheduleResource` and we refer to it as a `resource` for short. A scheduled session is represented by a `KalturaRecordScheduleEvent`. One resource can be used for many different events, but an event will only be associated with one resource. The room will be launched using an embed link that is made up of a Kaltura Session with specific configurations, all of which will be discussed below. 
+
+<img src="/Users/hunterp/Documents/GitHub/virtual-meeting-rooms/newrow_room.png" alt="newrow_room" style="zoom:40%;" />
```

Comment:
you can't use an image fro a local path

Reply:
"and is referred to as a 'resource' for short."

Try to not use words like: I, you, we, our

Also please put the bold part back

Reply:
This was fixed in 2nd commit in this pull request https://github.com/kaltura-vpaas/virtual-meeting-rooms/pull/3/commits/e43655ec017d32f010a21f1910e0d12e9b7993c4

Reply:
ok, see other comments above